tests/integration/test_fim/test_basic_usage/test_basic_usage_access_opened_files.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import random
+import string
+
+import pytest
+
+from wazuh_testing.fim import LOG_FILE_PATH, generate_params, check_time_travel
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+
+pytestmark = [pytest.mark.tier(level=0)]
+
+# Variables
+
+
+test_directories = [os.path.join(PREFIX, 'testdir1')]
+
+directory_str = ','.join(test_directories)
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
+testdir1 = test_directories[0]
+
+# configurations
+
+
+conf_params = {'TEST_DIRECTORIES': directory_str, 'MODULE_NAME': __name__}
+
+p, m = generate_params(extra_params=conf_params)
+
+configurations = load_wazuh_configurations(configurations_path, __name__, params=p, metadata=m)
+
+# Fixtures
+
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope='function')
+def create_and_restore_large_file(request):
+    if not os.path.exists(testdir1):
+        os.mkdir(testdir1)
+
+    file_size = 1024 * 1024 * 768   # 805 MB
+    chunksize = 1024 * 768
+    file_path = os.path.join(testdir1, 'large_file')
+    changed_path = os.path.join(testdir1, 'changed_name')
+
+    if os.path.exists(changed_path):
+        os.rename(changed_path, file_path)
+    elif not os.path.exists(file_path):
+        with open(file_path, "a") as f:
+            while os.stat(file_path).st_size < file_size:
+                f.write(random.choice(string.printable) * chunksize)
```

Comment:
This fixture should create and delete the file at the end of the test, so when we are finished, the environment is clean. This will create a new file for each test case, instead of renaming the file for the second test case. You can take a look at this example:

https://github.com/wazuh/wazuh-qa/blob/fe154ff2fcb38c70422fcaab31c0189025157bad/tests/integration/test_fim/test_files/test_report_changes/test_disk_quota_values.py#L53-L64

Reply:
Done in commit [d9cdee7](https://github.com/wazuh/wazuh-qa/pull/705/commits/d9cdee784e424ff2381297e4d686b6abb3e5bcb1)